fdbserver/worker.actor.cpp

```diff
@@ -1165,6 +1165,28 @@ struct SharedLogsValue {
 	  : actor(actor), uid(uid), requests(requests) {}
 };
 
+ACTOR Future<Void> chaosMetricsLogger() {
+
+	auto res = g_network->global(INetwork::enChaosMetrics);
+	if (!res)
+		return Void();
+
+	state ChaosMetrics* chaosMetrics = static_cast<ChaosMetrics*>(res);
+	chaosMetrics->clear();
+
+	loop {
+		wait(delay(FLOW_KNOBS->CHAOS_LOGGING_INTERVAL));
+
+		TraceEvent e("ChaosMetrics");
+		// double elapsed = now() - chaosMetrics->startTime;
+		double elapsed = timer_monotonic() - chaosMetrics->startTime;
```

Comment:
You can't use `timer_monotonic()` in simulation for anything that can become a network message, which is the case for `TrackLatest()` events.  Use `now()`.

Reply:
Replaced all timer_monotonic() calls to now()